### ISSUE TITLE
get_page_info error: Facebook served mbasic/noscript content unexpectedly on {response.url}

### ISSUE THREAD

**Issue by joshTheECE** [kevinzg/facebook-scraper#442]
Hi, I am trying to extract the about info of pages using the following lines of code:

`from facebook_scraper import get_page_info
get_page_info('CocoonsMillennialResorts', cookies='cookies.txt')
`

and gets the following error:

`UserWarning: Facebook served mbasic/noscript content unexpectedly on https://m.facebook.com/page_content_list_view/more/?page_id=102651475346745&start_cursor=%7B%22timeline_cursor%22:null,%22timeline_section_cursor%22:null,%22has_next_page%22:true%7D&num_to_fetch=4&surface_type=posts_tab
  f"Facebook served mbasic/noscript content unexpectedly on {response.url}"`

**Comment by neon-ninja** [comment]
This isn't an error, just a warning. Did the function return results?

**Comment by joshTheECE** [comment]
Yes it did return results. Thanks. However some do not have the about info.

**Comment by neon-ninja** [comment]
Which pages didn't return about info?

**Comment by l0rtk** [comment]
@neon-ninja i get this warnings and keeps showing on the console without stop,how can i stop a script if this warnings are non-stoping?

**Comment by neon-ninja** [comment]
CTRL-C should do it

**Comment by l0rtk** [comment]
> CTRL-C should do it

Yes but if i have script that's runs over the array of facebook id's and scrapes their profile posts what can i do then?

**Comment by neon-ninja** [comment]
This warning is safe to ignore, unless data are missing. Are you experiencing some other problem related to this warning?

**Comment by lorellav** [comment]
Hello, I have the same message and it does not return any data at all. Also trying to scrape posts from a group using get_posts. I also noticed that if "allow_extra_requests": True, it simply does not get anything, as in it executes the cell but does not get any data. thank you

**Comment by neon-ninja** [comment]
@lorellav Try pass cookies as per the readme
